qiskit/primitives/base/base_estimator.py

```diff
@@ -185,6 +119,63 @@ class BaseEstimatorV1(BasePrimitive, Generic[T]):
     """Estimator base class.
```

Comment:
```suggestion
    r"""Estimator base class.
```

Reply:
Wouldn't it be clearer to also mention the version in the docstring? (also applicable to sampler)

```suggestion
    r"""Estimator V1 base class.
```